guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,242 @@
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+**Fedimint** and **Cashu** are two unqiue bitcoin-backed ecash protocols. They differ in their approach to trust and governance. 
+
+---
+
+### [Cashu]({{ '/guide/how-it-works/ecash/cashu/' | relative_url }})
+
+Cashu lets users run and operate their own solo mints or place trust in a solo mint they choose to use.
+
+---
+
+### [Fedimint]({{ '/guide/how-it-works/ecash/fedimint' | relative_url }})
+
+Fedimint operates through a federated model where a group of trusted entities (called "guardians") manage funds collectively through the use of multi-sig.
+
+---
+
+## Using ecash
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/using-ecash.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/using-ecash@2x.jpg"
+   alt-text = "Surreal image featuring two people extending their arms from opposite sides, exchanging dollar bills in front of a retro computer. The computer, with a blank screen and an old keyboard, sits on a small desk. One person is dressed in a pink suit, while the other wears a yellow outfit, set against a vibrant pink and orange background."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+### Creating and redeeming ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/creation-and-redemption.png"
+   retina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/creation-and-redemption@2x.png"
+   alt-text = "Diagram illustrating the creation and redemption of ecash by Alice. On the left side, under ‘Creation,’ Alice sends 100 sats to an ecash mint, which creates ecash tokens, and Alice receives 100 sats in ecash form. On the right side, under ‘Redemption,’ Alice sends 100 sats in ecash form to the mint, which burns the ecash tokens, and Alice receives 100 sats back in Bitcoin. The process flow is represented with dashed arrows and labeled steps."
+   caption = "Alice mints 100 sats into ecash through a mint and later redeems the ecash tokens to reclaim her original 100 sats in bitcoin."
+   width = 800
+   height = 508
+   layout = "full-width"
+%}
+
+Cashu and Fedimint use various terms for the processes of creating, redeeming, and transferring bitcoin-backed ecash. To provide clarity, we'll use neutral terms in our explanations and provide a translation table for specific implementations.
+
+#### Creating ecash
```

Comment:
Might be worth mentioning that, although ecash works differently under the hood, the end user experience for ecash wallets is very similar to Lightning wallets. Users send and receive sats, with few notable differences to Lightning. But the overall experience is very similar. For example, users send sats to a wallet and don't really notice that the bitcoin gets converted to ecash, and vice versa.

Reply:
Great suggestion. I updated this to include this:

_To provide clarity, we’ll use neutral terms in our explanations and provide a translation table for specific implementations. While the underlying technology differs, the end user experience of ecash wallets is very similar to Lightning wallets - users simply send and receive sats without needing to understand the technical conversion between bitcoin and ecash happening behind the scenes. Most users won’t notice any significant differences from their usual Lightning wallet experience._